.github/workflows/ci.yaml

```diff
@@ -104,7 +104,14 @@ jobs:
       run: |
         cd nitro-testnode
         ./test-node.bash --detach
-    
+        
+    - name: Setup avalanche-cli node
+      if: ${{matrix.crate == 'rosetta-testing-avalanche'}}
+      run: |
+        docker pull manojanalog/avalanche-cli-analog
+        docker run -v MY_LOCAL_CLI_DIR:/root/.avalanche-cli/ manojanalog/avalanche-cli-analog blockchain create localnew --evm --evm-token SUB --genesis ./genesis.json --teleporter=false --vm-version v0.6.9 
```

Comment:
Can you include this docker image on `./script/pull_nodes.sh`? btw why this cli is in your repo? can't we move it to the official analog repo?

Reply:
for puting it in analog repo let me talk to @penumbra23

Reply:
Ok, btw where is the Dockerfile for the `avalance-cli-analog`?

Reply:
updated docker image from manojanalog to analoglabs